src/generator/xmlAdditionalProps.ts

```diff
@@ -26,22 +26,31 @@ type additionalProperties map[string]*string
 // UnmarshalXML implements the xml.Unmarshaler interface for additionalProperties.
 func (ap *additionalProperties) UnmarshalXML(d *xml.Decoder, start xml.StartElement) error {
```

Comment:
We should adopt the implementation from track 1.5 (unsure why I didn't do this to begin with).

https://github.com/Azure/azure-storage-blob-go/blob/master/azblob/zz_generated_models.go#L38

Reply:
I did take a look at the implementation for track 1.5 it doesnt unmarshal the content for BlobMetadata properly. The XML could be malformed though. Since it for the additional properties the startElement is empty for some reason.